README.md

```diff
@@ -49,24 +49,25 @@ Work | Don’t work
 **Mancy** |
 **Meld** |
 **Mousepad** |
-**Notepad** (Win) |
+**Notepad** (Windows) |
 **QtCreator** | 
 **RStudio** ([instructions](https://github.com/tonsky/FiraCode/wiki/RStudio-instructions)) |
 **Scratch** |
 **TextAdept** (Linux, Mac) |
+**TextEdit** (Mac OS X text editor) |
 **TextMate 2** |
 **Visual Studio 2015** |
 **Visual Studio Code** ([instructions](https://github.com/tonsky/FiraCode/wiki/VS-Code-Instructions)) |
 **Xamarin Studio/Monodevelop** |
 **Xcode** (with [this plugin](https://github.com/robertvojta/LigatureXcodePlugin)) |
-Probably work: **Smultron, Vico** |
+Probably works: **Smultron, Vico** |
 
 ### Browser support
 
-- IE 10+, Edge: enable with `font-feature-settings: "calt" 1;`
+- IE 10+ and Edge: enable with `font-feature-settings: "calt" 1;`
 - Firefox
 - Safari
-- Chromium-based browsers: enable with  `font-variant-ligatures: contextual;` or `text-rendering: optimizeLegibility` (see [571246](https://code.google.com/p/chromium/issues/detail?q=font-variant-ligatures&id=571246&thanks=571246&ts=1450553433&))
+- Chromium-based browsers (like Chrome, most Android browsers, and Opera): enable with  `font-variant-ligatures: contextual;` or `text-rendering: optimizeLegibility` (see [bug 571246](https://code.google.com/p/chromium/issues/detail?q=font-variant-ligatures&id=571246&thanks=571246&ts=1450553433&))
```

Comment:
most Android browsers? I don't think that Firefox and Dolphin are Chromium-based.